ansible/roles/digidecs/templates/digidecs.json.j2

```diff
@@ -0,0 +1,12 @@
+{
+  "server": {
```

Comment:
You mention this hardcoded port multiple times in ansible. I suggest moving it to a variable